epinorm/norm.py

```diff
@@ -122,6 +129,86 @@ def get_feature_ids(self):
             .dropna()
             .unique()
         )
+    
+    def _admin_levels_table(self):
+        """
+        This method retrives the information in the "countries.csv" and "administrative_units.tsv".
+        It returns a dictionnary that maps a country name (from countries.csv) to a list of its administrative levels.
+        administrative levels are encoded as dictionaries with properties:
+        {
+            name,
+            admin_level,
+            osmId
+        }
+
+        it also tranliterates information from the administrative_units.tsv file.
+        """
+
+        # * create a mapping from country name to its two letter code
+        country_to_code = {}
+        df = pd.read_csv(COUNTRIES_DATA)
+        for _, row in df.iterrows():
+            country_to_code[row["name"]] = row["alpha_2"]
```

Comment:
Here we can use `.to_dict()` to avoid a `for` loop.

```suggestion
        # Create a mapping from country name to its two letter code.
        country_to_code = pd.read_csv(COUNTRIES_DATA, index_col="name")["alpha_2"].to_dict()
```